README.md

```diff
@@ -1,7 +1,54 @@
-# Fivetran Cloud Functions
+# Fivetran
 
-[Cloud functions](https://fivetran.com/docs/functions) for ETL with serverless computing platforms like [AWS Lambda](https://aws.amazon.com/lambda/), [GCloud Functions](https://cloud.google.com/functions/) and [Azure Functions](https://azure.microsoft.com/Functions/Serverless).
+[Fivetran](https://fivetran.com/) helps you centralize data from disparate sources into your data storage platform that you can manage directly from your browser. Fivetran’s fully automated connectors extract your data from cloud applications, databases, event logs, and more, and load the data into your cloud warehouse, database, or data lake.
+
+## Fivetran Cloud Function Connectors
+
+[Function connectors](https://fivetran.com/docs/functions) allow you to code a custom data connector as an extension of Fivetran. If you have a custom data source or a private API, use our Function connectors to build robust serverless ELT data pipelines. You only have to write the cloud function to extract the data from your source. Fivetran will load and transform the data in your destination. 
```

Comment:
>Fivetran will load and transform the data in your destination

I think we should just say "Fivetran will load the data in your destination"

>If you have a custom data source or a private API

There can be more use-cases. maybe we can say "For example, If you have a custom data source or a private API"

Reply:
@fivetran-shashanktripathi I re-used our already approved content from the Functions page. Here is a screenshot for your reference. 
![image](https://user-images.githubusercontent.com/60129976/120595681-6aa47780-c460-11eb-8fd3-ea9752878241.png)

Reply:
@fivetran-abhijeet I think as a follow-up, we should apply the suggestion "For example, If you have a custom data source or a private API" both here and in the Functions overview doc to make it clear there could be other use cases. Underlining the open-ended nature of function connectors seems important.

But, this can be a tiny followup rather than holding us up here.

Reply:
sure @AK-Fivetran. I will create a follow-up.